### DISCUSSION TITLE
Pagination uses wrong path to generate url

### DISCUSSION THREAD

**Discussion by faltjo** [livewire/livewire#6705] [7. Bugs (archived)]
### Livewire version

3.0.2

### Which PHP version are you using?

PHP 8.2

### Repository URL

https://github.com/faltjo/livewire-pagination

### Steps To Reproduce

1. Use Livewire's pagination feature (https://livewire.laravel.com/docs/pagination) alongside Laravel Eloquent's paginator.
2. The paginator returns an unexpected path, especially after a Livewire update it does point to `.../livewire/update` instead of `.../yourpage`.
3. When using the urls of the paginator (for example in a custom pagination view `$paginator->url($somePageNumber)`) the links break. 

### Minimal test case
I tried to come up with a minimal test case to reproduce this behaviour:
1. clone https://github.com/faltjo/livewire-pagination
4. `composer install`
5. `cp .env.example .env`
6. `php artisan key:generate`
7. `./vendor/bin/pest --init`
8. `./vendor/bin/pest`

In the test case the returned paginator provides the expected items and other attributes, except for `path` and `options`.
I tried to overwrite the path using `setPath` or `withPath` on the paginator, but the test still does not pass.

### Additional information

When building the url the paginator is using its path: https://github.com/illuminate/pagination/blob/master/AbstractPaginator.php#L184

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by EchedelleLR** [comment]
This is still an issue.

The solution someone found was this other one: https://ghanshyamashra.online/mastering-livewire-pagination-in-laravel-fix-common-issues-like-a-pro/

**Reply by EchedelleLR** [reply]
And yes, is what you actually tried...
